src/components/FeatureCards/FeatureCards.tsx

```diff
@@ -47,7 +47,21 @@ export const FeatureCards = () => {
 
   const slicedFeatureCards = useMemo(() => {
     if (Array.isArray(cards) && !!cards.length) {
-      return cards
+      // Remove excluded feature cards
+      const excludedFeatureCards = cards?.filter((el, index) => {
+        if ((el.attributes.featureCardsExclusions?.data || []).length === 0) {
+          return true;
+        }
+
+        const exclusions =
+          el.attributes.featureCardsExclusions?.data.map(
+            (item) => item.attributes.uid,
+          ) ?? [];
```

Comment:
why this "?? []" check if you already look for the `el.attributes.featureCardsExclusions?.data` to be not null in the if condition above?